extensions/query-history/README.md

```diff
@@ -2,6 +2,40 @@
 
 Adds a Query History panel for viewing and running past executed queries.
 
+### How do I view the history?
+
+Query History is displayed as a tab in the tab panel, which is toggled by the *View: Toggle Panel* command.
+
+![Query History tab](https://github.com/microsoft/azuredatastudio/tree/master/extensions/query-history/images/QueryHistoryTab.png)
+
+Initially this view will be empty but once you execute a query editor that will be captured in the window - with a separate row displayed for every query executed.
+
+![Query History tab with queries](https://github.com/microsoft/azuredatastudio/tree/master/extensions/query-history/images/QueryHistoryTabWithQueries.png)
+
+Each row consists of 3 parts :
+	1. Status icon : This will be a ✔️ if the query executed successfully. If any errors occured a ❌is shown.
```

Comment:
occurred

Reply:
Well that's what I get for not having my spellcheck extension on my laptop...thanks!